CriteoPublisherSdk/Sources/Configuration/CR_ConfigManager.m

```diff
@@ -40,7 +41,8 @@ - (instancetype)initWithApiHandler:(CR_ApiHandler *)apiHandler
 }
 
 - (void)refreshConfig:(CR_Config *)config {
-  [self->apiHandler getConfig:config
+  CR_RemoteConfigRequest *remoteConfigRequest = [CR_RemoteConfigRequest createFromConfig:config];
```

Comment:
Unless there are other request objects around just naming this var `request` is enough (same goes for the whole commit)

Reply:
done